ida-customizations/ida-course/discord-info.md

```diff
@@ -30,21 +30,27 @@ If you have any problems, send us an [email](mailto:academy@interchain.io).
 
 Within the Academy Discord, you will find the following topic-specific channels:
 
-![IMAGE DESCRIPTION](/ida-course/images/discord-channel.png)
+![IMAGE DESCRIPTION](/ida-course/images/discord-ida-c3-channels.jpg)
 
-Please post your question in the channel that fits best. This will allow you and the support team to keep track of your question. The channels correspond to the main topics covered in the Academy content. You will find a description of each channel on Discord.
+* We will share regular program updates via the **c3-announcements-and-organization** channel. Make sure to check it regularly.
+* The **c3-lounge** channel is dedicated to off-topic discussions. Introduce yourself and feel free to relax and chat about your favorite TV shows, films, and books, share pet photos, and anything else that comes to mind!
+* You should ask all content-related questions within the **c3-support forum** channel. There you will find tags which correspond to the main topics covered in the Academy content. These will allow you and the support team to keep track of your queries.
 
 ## How to ask a question
 
 You can post a message with your question directly into any channel.
 
 ![IMAGE DESCRIPTION](/ida-course/images/discord-question.png)
 
-Make sure you mark your message as a question by adding the question emoticon (?) to it. This will create a new support ticket.
+When posting a question in the **c3-support-forum**, you need to add a title and select the tag of the topic your question is related to. 
+
+![IMAGE DESCRIPTION](/ida-course/images/discord-howto-supportforum.jpg)
+
+Make sure you mark your message as a question by adding the question emoticon (?) to it **_as a react_**. This will create a new support ticket.
 
 ![IMAGE DESCRIPTION](/ida-course/images/discord-question-add.png)
 
-Once a ticket has been created, tutors and expert instructors are informed about the new question and can reply directly to your original post. You will also receive a DM from _Trisha | Support Bot_, our suppor tbot, which confirms that a ticket has been opened.
+Once a ticket has been created, tutors and expert instructors are informed about the new question and can reply directly to your original post. You will also receive a DM from _Tricia | Support Bot_, our support bot, which confirms that a ticket has been opened.
```

Comment:
Do we really want to say "from _Tricia | Support Bot_, our support bot"?

(see also line 61)